ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -810,45 +812,41 @@ private void shareCurrentTract() {
         }
     }
 
+    /**
+     * This method takes in a Manifest Object and return a shareable Like for
+     *  the corresponding tool.
+     * @param manifest = The Manifest Object to be used.
+     * @return = Url string of the shareable link.
+     */
     @NonNull
     private String buildSharingURL(@NonNull final Manifest manifest) {
         final Uri.Builder uri;
-        switch (manifest.getCode()) {
-            // temporary cases until knowgod.com is rebuilt
-            case "honorrestored":
-                uri = Uri.parse("https://godtoolsapp.com").buildUpon();
-                break;
-            case "thefour":
-                uri = Uri.parse("https://thefour.com").buildUpon();
-                break;
-            default:
-                uri = URI_SHARE_BASE.buildUpon()
-                        .appendPath(LocaleCompat.toLanguageTag(manifest.getLocale()).toLowerCase())
-                        .appendPath(getCodeForShareActivity(manifest))
-                        .appendPath(getPagePathPartForSharing(manifest));
-        }
+
+        uri = URI_SHARE_BASE.buildUpon()
+                .appendPath(LocaleCompat.toLanguageTag(manifest.getLocale()).toLowerCase())
+                .appendPath(getCodeForShareActivity(manifest))
+                .appendPath(getPagePathPartForSharing());
```

Comment:
`getCodeForShareActivity` and `getPagePathPartForSharing` can both be consolidated into the `buildSharingURL` method. there isn't any clear benefits for them remaining as separate methods

Reply:
On it.